modules/gitops-accessing-the-gitops-operator-metrics.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/monitoring-the-gitops-operator-performance.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-accessing-the-gitops-operator-metrics_{context}"]
+= Accessing the {gitops-shortname} Operator metrics
+
+You can access the Operator metrics from the *Administrator* perspective of the {OCP} web console to track the performance of the Operator.
+
+.Prerequisites
+
+* You have access to the {OCP} web console.
+* The {gitops-title} Operator is installed in the default `openshift-gitops-operator` namespace.
+* The cluster monitoring is enabled on the `openshift-gitops-operator` namespace.
```

Comment:
After doing some testing of my own I think the only restrictions for metrics to show up are that
- the operator must be installed in a namespace starting with `openshfit-` like `openshfit-xyz`
- cluster monitoring should be enabled